src/include/libreport_types.h

```diff
@@ -25,90 +25,67 @@
 typedef gchar **string_vector_ptr_t;
 typedef const gchar *const *const_string_vector_const_ptr_t;
 
-#define string_vector_new_from_string libreport_string_vector_new_from_string
-string_vector_ptr_t string_vector_new_from_string(const char *vector);
-#define string_vector_free libreport_string_vector_free
-void string_vector_free(string_vector_ptr_t vector);
+string_vector_ptr_t libreport_string_vector_new_from_string(const char *vector);
```

Comment:
Unrelated, but all of this code needs to eventually be shot into the Sun, so no trace remains.